### DISCUSSION TITLE
Standup (Monday, November 14th)

### DISCUSSION THREAD

**Discussion by DavBot02** [BBGTC/food-bank-app#61] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished #53 (minus the qr)
 - Read about stack navigator's header customization

**What are your priorities for the next 24 hours?**:
 - Work on the qr
 - Migrate @BrandonMagana 's header 

**Blockers**:
 - None

**Shoutouts**:
 - @BrandonMagana because he makes beautiful screens

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - TBH, nothing

**What are your priorities for the next 24 hours?**:
 - Create axios client
 - Resolve paths
 - Develop backend for priority list

**Blockers**:
 - Swift homework
 - Broke my laptop

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Opened #60 

**What are your priorities for the next 24 hours?**:
 - Planning the week

**Blockers**:
 - I was a little busy with homework.
 - Actividad integradora de algos

**Shoutouts**:
 - @everyone you guys are great

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished homescreen components 

**What are your priorities for the next 24 hours?**:
 - Help @CarlosCRG19  connect backend with frontend
 - Planning for the week

**Blockers**:
 - Pending homeworks 
 - Going to TJ on thursday

**Shoutouts**:
 -  @antonieto for stop being baquetón and reviewed my PR
